docs/sphinx/conf.py

```diff
@@ -149,6 +149,7 @@
     "mlflow",
     "mysql",
     "oauth2client",
+    "orjson",
```

Comment:
is this a related change?

Reply:
Related enough-- has to do with docs build, which started failing on this PR due to this new dep.